app/assets/stylesheets/v3/layout_components/side_nav.scss

```diff
@@ -1,11 +1,29 @@
+/* Desktop Header/Side-Nav */
+
 .side-nav {
+  @include grid-column(12,0);
   background: $white;
-  @include grid-column(2,0);
+  box-shadow: $base-box-shadow;
+  height: $side-nav-height;
+  position: fixed;
```

Comment:
I noticed the marketing site's navigation isn't fixed. How did you decide to make ours that way?

Reply:
Their main navigation isn't fixed, but their sub-navigation is (The blue one). I guess I figured that we rely pretty heavily on a member having quick access to our side nav (now top nav) and it seemed reasonable to replace one sticky navigation with another?

Reply:
sounds reasonable to me!